Vagrantfile

```diff
@@ -116,7 +117,9 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
               consul_join: consul_join,
               consul_retry_join: consul_retry_join,
               marathon_master_peers: marathon_master_peers,
-              marathon_zk_peers: marathon_zk_peers
+              marathon_zk_peers: marathon_zk_peers,
+              mesos_master_quorum: 2,
```

Comment:
do you mean those two? because rest of them are dynamic
mesos_master_quorum: 2
consul_bootstrap_expect: 3

But in my opinion those two should be in a vagrant.yml file